core/src/main/java/org/apache/sdap/mudrod/main/MudrodEngine.java

```diff
@@ -138,48 +138,37 @@ private InputStream locateConfig() {
       LOG.info("Loaded config file from " + configFile.getAbsolutePath());
       return configStream;
     } catch (IOException e) {
-      LOG.info("File specified by environment variable " + MudrodConstants.MUDROD_CONFIG + "=\'" + configLocation + "\' could not be loaded. " + e.getMessage());
+      LOG.info("File specified by environment variable " + MudrodConstants.MUDROD_CONFIG + "=\'" + configLocation + "\' could not be loaded. Default configuration will be used." + e.getMessage());
```

Comment:
Please change to and use [paramaterized logging](https://www.slf4j.org/faq.html#logging_performance) for all future logging. The implementation should be 
```
LOG.info("File specified by environment variable {} = '{}' could not be loaded. Default configuration will be used.", MudrodConstants.MUDROD_CONFIG, configLocation, e.getMessage());
```
Please use paramaterized logging whenever possible, it is a much more efficient implementation than string concatenation